template_feed/Microsoft.DotNet.Common.ItemTemplates/content/BuildProps/.template.config/template.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [
+    "MSBuild",
+    "props"
+  ],
+  "name": "MSBuild Directory.Build.props file",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "An empty Directory.Build.props file which can be used to specify MSBuild properties for an entire folder",
+  "tags": {
+    "type": "item",
+    "language": "MSBuild"
+  },
+  "sources": [
+    {
+      "source": "./",
+      "target": "./"
+    }
+  ],
+  "identity": "Microsoft.Standard.QuickStarts.DirectoryProps",
+  "shortName": "buildprops",
+  "symbols": {
+    "inherit": {
+      "type": "parameter",
+      "datatype": "bool",
+      "defaultValue": "false",
+      "defaultIfOptionWithoutValue": "true",
+      "displayName": "inherit",
```

Comment:
`display name` will be used by IDE (VS). It's better to use sentence case here, may be just "inherit" is not ok for this purpose.